server/src/test-data/fixtures.js

```diff
@@ -38,7 +38,7 @@ export const USERS = [
     id: DEFAULT_USER_ID,
     username: DEFAULT_USERNAME,
     password: 'test',
-    name: 'Paddy Platypus',
+    displayName: 'Paddy Platypus',
```

Comment:
you updated this one but not the rest...

Reply:
Others can default :p fuck em!

Reply:
Done.